src/builder_basics.rs

```diff
@@ -0,0 +1,41 @@
+//! # Builder Basics Sample
+//!
+//! This sample demonstrates how to use the builder with a simple glade file
+
+extern crate glib;
```

Comment:
We don't need this import, do we? Let's not confuse anyone who might cargo-cult this into their own projects :)